deploy/gk-deploy

```diff
@@ -233,6 +233,7 @@ while [[ $# -ge 1 ]]; do
         ;;
         -abort)
         ABORT=1
+        SKIP_PREREQ=1
```

Comment:
I just think If we use SKIP_PREREQ for something else tomo then this will impact Abort as it always uses it.